assets/global.js

```diff
@@ -846,11 +846,19 @@ class VariantSelects extends HTMLElement {
         const html = new DOMParser().parseFromString(responseText, 'text/html')
         const destination = document.getElementById(`price-${this.dataset.section}`);
         const source = html.getElementById(`price-${this.dataset.originalSection ? this.dataset.originalSection : this.dataset.section}`);
+        const inventorySource = html.getElementById(`inventory-${this.dataset.originalSection ? this.dataset.originalSection : this.dataset.section}`);
+        const inventoryDestination = document.getElementById(`inventory-${this.dataset.section}`);
+
         if (source && destination) destination.innerHTML = source.innerHTML;
+        if (inventorySource && inventoryDestination) inventoryDestination.innerHTML = inventorySource.innerHTML;
 
         const price = document.getElementById(`price-${this.dataset.section}`);
+        const inventory = document.getElementById(`inventory-${this.dataset.section}`);
 
         if (price) price.classList.remove('visibility-hidden');
+        if (inventory && inventorySource == null) inventory.classList.add('hidden');
+        if (inventory && inventorySource !== null) inventory.classList.remove('hidden');
```

Comment:
Hey, i think it would be better to remove all the white space when a block disappears (even for price) but i agree we can just keep the current logic for now (keeping the white space like for prices). Let's just fix that issue later, for every block.